apps/serdes/serialize_test.cpp

```diff
@@ -0,0 +1,25 @@
+#include "Deserializer.h"
+#include "Serializer.h"
+#include <Halide.h>
+using namespace Halide;
+
+// a simple round trip test
```

Comment:
What if we had an IR comparator to validate the deserialized pipeline matches the original in-memory pipeline?

Reply:
See `IRComparator.cpp`